srml/democracy/src/lib.rs

```diff
@@ -218,6 +235,9 @@ decl_storage! {
 		/// voter when called with the referendum (you'll get the default `Vote` value otherwise). If you don't want to check
 		/// `voters_for`, then you can also check for simple existence with `VoteOf::exists` first.
 		pub VoteOf get(vote_of): map (ReferendumIndex, T::AccountId) => Vote;
+
+		/// Get the account (and lock periods) to which another account is delegating vote.
+		pub Delegations get(delegations): linked_map T::AccountId => (T::AccountId, LockPeriods, ReferendumIndex);
```

Comment:
Since it's a map it supports only one `ReferendumIndex` anyway, no? Why do we need it at all? It seems that it might be just populated with old referendas

Reply:
I removed it, I'm not sure users want to delegate for a specific referendum, it seems too fine grained.